### ISSUE TITLE
Other gamemodes than standard?

### ISSUE THREAD

**Issue by jamuwu** [Francesco149/oppai#26]
Just wondering if you were ever planning on supporting any other modes, like mania.

**Comment by TheMeq** [comment]
Yes plz

**Comment by Francesco149** [comment]
yes, but only when the difficulty calculation code for said modes is officially public. osu!lazer currently only has osu diff calc

**Comment by Francesco149** [comment]
taiko diff calc is now in osu lazer, but it will take a while to implement it into oppai as converting beatmaps to taiko requires slider curve calculations... dunno when I'll feel like it

**Comment by Francesco149** [comment]
actually I think we can cheat and avoid them entirely, I'll let you know

EDIT: yeah seems like it shouldn't affect the difficulty calculation much, if at all. well it'll still take a while before I feel like it

**Comment by jamuwu** [comment]
Any updates on your motivation to do it?

**Comment by thehowl** [comment]
Just an update to those stumbling upon this thread: Taiko PP is implemented! https://github.com/Francesco149/oppai-ng/commit/9eca9d58a29c46bb3e7b938852bf7fb524e886f8
